en/reference/schema-reference.html

```diff
@@ -1283,6 +1284,18 @@ <h2 id="alias">alias</h2>
 </p>
 
 
+<h2 id="dictionary">dictionary</h2>
+<p>Contained in <code><a href="#field">field</a></code>.
+Specifies details of the dictionary. Applies only to <a href="#attribute">attributes</a> annotated with <code>fast-search</code>.
+You can specify either <code>btree</code> or <code>hash</hash>, or both.</p>
+<p>
+Note that prefix search for strings and range search for numeric fields will fallback to full scan if using <code>hash</code>.
+It is primarily intended for use when you many unique terms that have short postinglists, where the dictionary lookup cost is significant.
+Normally <code>btree</code> is you best choice. This is also the default.<p>
+<p>
+In addition you can specify if you want a folded dictionary, <code>uncased</code>, or <code>cased</code> for string attributes.
```

Comment:
I don't think the term _folded dictionary_ has been used before. Can this be explained in a simple way instead?

Reply:
Done.